.github/component-label-map.yml

```diff
@@ -2,10 +2,18 @@ blog:
   - changed-files:
       - any-glob-to-any-file:
           - content/en/blog/**
-registry:
+docs:
   - changed-files:
       - any-glob-to-any-file:
-          - data/registry/**
+          - content/en/**
+docs:
+  - changed-files:
+      - any-glob-to-any-file:
+          - content/en/**
+translation:
```

Comment:
How about:
```suggestion
i18n:
```

Reply:
then it would be l10n from my understanding. I am also not happy with "translation", I also was thinking about "lang"

Reply:
Right, `l10n`. Btw, I think that the rule should be

```yaml
  - changed-files:
      - any-glob-to-any-file:
          - content/**
          - '!content/en/**'
```